omegatrade-app/backend/app/models/simulation.model.js

```diff
@@ -0,0 +1,62 @@
+'user strict';
+const database = require('./../config/database.js');
+const { v4: uuidv4 } = require('uuid');
+const Simulation = function () { };
+
+Simulation.getAll = async function () {
+    const [result] = await database.run({
+        sql: `SELECT sml.sId as sId,sml.companyId as companyId, sml.status as status, cy.companyName as companyName, cy.companyShortCode as companyShortCode FROM simulations sml 
+              LEFT JOIN companies cy ON sml.companyId = cy.companyId 
```

Comment:
This query should use `INNER JOIN` instead of `LEFT JOIN`, as there should never be a simulation without a company.

Reply:
This issue has been resolved